components/script/dom/htmlscriptelement.rs

```diff
@@ -647,7 +649,7 @@ impl HTMLScriptElement {
 
                     fetch_inline_module_script(
                         ModuleOwner::Window(Trusted::new(self)),
-                        text.clone(),
+                        Rc::new(text.clone()),
```

Comment:
Can we use the earlier Rc that was created instead of making a new one here?

Reply:
I'm guessing yes; I was just trying to get the compiler to stop yelling at me :)

I'll look into it tomorrow

Reply:
Okay, I just did it!